kubecost-exporter.md

```diff
@@ -0,0 +1,117 @@
+    ```
+
+    > If you want to use a namespace other than `cost-model`, you will have to edit the `ClusterRoleBinding` after applying the YAML to change `subjects[0].namespace`. You can do this with `kubectl edit clusterrolebinding cost-model`.
+
+2. To verify that metrics are available:
+
+    ```
+    kubectl port-forward --namespace cost-model service/cost-model 9003
+    ```
+
+    Visit [http://localhost:9003/metrics](http://localhost:9003/metrics) to see exported metrics
+
+Add Kubecost scrape config to Prom ([more](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus))
+```
+- job_name: cost-model
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+    - targets: [‘cost-model.cost-model.:9003/metrics’]
+```
+
+Done! Kubecost is now exporting cost metrics. See the following sections for different metrics available and query examples.
+
+## Available Prometheus Metrics 
+
+| Metric       | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| node_cpu_hourly_cost | Hourly cost per vCPU on this node  |
+| node_gpu_hourly_cost | Hourly cost per GPU on this node  |
+| node_ram_hourly_cost   | Hourly cost per Gb of memory on this node                       |
+| node_total_hourly_cost   | Total node cost per hour                       |
+| container_cpu_allocation   | Average number of CPUs requested over last 1m                      |
+| container_memory_allocation_bytes   | Average bytes of RAM requested over last 1m                 |
+
+
+By default, the cost metrics are based on public billing APIs. Supported platforms: AWS, Azure, and GCP. For on-prem clusters, prices are based on configurable defaults. 
+
+More metrics are available in the recommended install path and are described in [PROMETHEUS.md](PROMETHEUS.md).
+
+## Dashboard examples
+
+Here’s an example dashboard using Kubecost metrics: 
+
+![sample dashboard](https://grafana.com/api/dashboards/8670/images/5480/image)
+
+You can find other example dashboards at https://grafana.com/orgs/kubecost
+
+## Example Queries
+
+Once Kubecost’s cost model is running in your cluster and you have added it in your Prometheus scrape configuration, you can hit Prometheus with useful queries like these:
+
+#### Monthly cost of all nodes
+
+```
+sum(node_total_hourly_cost) * 730
+```
+
+#### Hourly cost of all load balancers broken down by namespace
+
+```
+sum(kubecost_load_balancer_cost) by (namespace)
+```
+
+#### Monthly rate of each namespace’s CPU request
+
+```
+sum(container_cpu_allocation * on (node) group_left node_ram_hourly_cost) by (namespace) * 730
+```
+
+#### Historical spend for all `fluentd` pods in the `kube-system` namespace
```

Comment:
This looks like its actually historical _memory_ spend. Is that right @dwbrown2?

Reply:
That's correct! Thanks for catching.

For these initial queries, I just wanted to keep it simple... as opposed to showing mem + cpu + loadbalancer, etc etc.